src/main/java/com/bp3/camunda/camunda7/C7RESTConnector.java

```diff
@@ -79,18 +97,44 @@ public void execute(ExternalTask externalTask, ExternalTaskService externalTaskS
         setPayload(request, httpMethod, httpPayload);
 
         // call the REST service
-        HttpResponse response = request.execute();
+        try {
+            HttpResponse response = request.execute();
+
+            // set the output variable
+            log.debug("RESPONSE: {}", response.getResponse());
+            VariableMap variables = Variables.createVariables();
+            if (outputVariableName != null) {
+                variables.putValue(outputVariableName, response.getResponse());
+            }
 
-        // set the output variable
-        log.debug("RESPONSE: {}", response.getResponse());
-        VariableMap variables = Variables.createVariables();
-        if (outputVariableName != null) {
-            variables.putValue(outputVariableName, response.getResponse());
+            // complete the external task
+            externalTaskService.complete(externalTask, variables);
+        } catch(ConnectorException e) {
+            log.error("CONNECTOR_ERROR: {}", e.getLocalizedMessage(), e.fillInStackTrace());
+
+            if (Objects.nonNull(errorHandlingMethod)) {
```

Comment:
Same as a above, intended for filter predicate usage
```suggestion
            if (errorHandlingMethod != null) {
```

Reply:
Updated